File: src/main/java/com/example/GradeClassifier.java
package com.example;

public class GradeClassifier {

    public String classify(int[] scores) {
        if (scores == null || scores.length == 0) {
            throw new IllegalArgumentException("Danh sách điểm không được rỗng");
        }

        int total = 0;
        for (int score : scores) {
            total += score;
        }

        double average = (double) total / scores.length;

        if (average >= 85) {
            return "Giỏi";
        } else if (average >= 70) {
            return "Khá";
        } else if (average >= 50) {
            return "Trung bình";
        } else {
            return "Yếu";
        }
    }
}

File: grade_classifier.py
def classify(scores):
    """Tính điểm trung bình và phân loại học lực."""
    if not scores:
        raise ValueError("Danh sách điểm không được rỗng")

    total = 0
    for score in scores:
        total += score

    average = total / len(scores)

    if average >= 85:
        return "Giỏi"
    elif average >= 70:
        return "Khá"
    elif average >= 50:
        return "Trung bình"
    else:
        return "Yếu"
